lms/djangoapps/program_enrollments/api/linking.py

```diff
@@ -145,6 +171,36 @@ def _get_lms_users(lms_usernames):
     }
 
 
+def _unlink_course_enrollments_from_program_course_enrollment(program_enrollment):
```

Comment:
I think I'll change this to `unlink_program_enrollment`, since it operates on more than just the course enrollments now.

Reply:
Yeah, changing the function name to `unlink_program_enrollment` make sense.